### DISCUSSION TITLE
Visualize multiple unit cells

### DISCUSSION THREAD

**Discussion by mbocus** [scanberg/viamd#77] [Ideas]
Hello everyone! 

I was wondering whether it would be (or maybe already is) possible to visualize multiple unit cells for a periodic structure.
Moreover, it would be also nice (for me at least) if one could specify the wrapping center when applying the PBC command. 

Thanks in advance!
Massimo

**Comment by mathieulinares** [comment]
Dear Massimo. 
Thanks for your comment. It is not possible to visualize multiple cells at the moment. We are planning to implement that in the future. ANd will update (and close) this discussion when it's done.
Regarding your second comment. I am not sure to understand what you want to do. Do you know that you can recentered your trajectory on a selection by right clicking on it. and choosing Recenter Trajectory.. > On Selection. Would it solve your issue!?

**Reply by mbocus** [reply]
Dear Mathieu,

Thanks for your reply and for the great work you guys are doing! I'm looking forward to see the multiple unit cells feature implemented. As for the second comment, yes, that is exactly what I was looking for, many thanks for the tip!

Best,
Massimo

**Reply by mathieulinares** [reply]
You are welcome! Thanks for the encouragement. Here is a video where it is explained in more detail: https://www.youtube.com/watch?v=o7KxGW19Qjk 
I will leave this ticket open until we implement the multiple cell. 
Cheers.
Mathieu
